src/services/utilities.ts

```diff
@@ -419,6 +419,11 @@ namespace ts {
     }
 
     export function positionBelongsToNode(candidate: Node, position: number, sourceFile: SourceFile): boolean {
+        if (candidate.kind === SyntaxKind.IfStatement) {

```

Comment:
this does not seem like the right place for this change, i will have to go and look into the smart indenter to find a better fix though.